docs/leo-rover/leo-examples/follow-marker.mdx

```diff
@@ -0,0 +1,263 @@
+```
+
+Then, you need to source the directory and build the packages:
+
+```bash
+cd ~/ros_ws
+source install/setup.bash
+colcon build
+```
+
+:::info
+
+If your installation went without any errors, then you have successfully
+installed required software.
+
+:::
+
+## Examples
+
+To run the example, you need to be connected to the rover via ssh. Once you do
+this, type in the terminal:
+
+```bash
+ros2 launch leo_example_follow_aruco_marker follow_aruco_marker.launch.xml
+```
+
+The rover will start following the marker once it detects it.
+
+:::warning
+
+By default the rover follows only the marker with id 0, and has marker tracker
+configured for a marker of size 15cm. If your printed marker doesn't comply with
+the requrements you need to follow the configuration instructions.
+
+:::
+
+### Configuration
+
+All nodes launched in the example have configurable parameters, therefore it's
+easy to adjust the system for your conditions. To be able to change anything you
+need to have ROS installed on your computer and be connected to the rover's
+network.
+
+If you don't have ROS installed, you can follow this guide:
+
+<LinkButton
+  to="/leo-rover/advanced-guides/install-ros-on-your-computer"
+  title="Install ROS on your computer"
+  description="Learn how to install the Robot Operating System (ROS) on your computer.
+  Step-by-step guide for beginners."
+/>
+
+Beeing connected and having ROS installed, next you have to source the `ROS`
+workspace:
+
+```bash
+source /opt/ros/${ROS_DISTRO}/setup.bash
+```
+
+Now just start the `Dynamic Reconfigure` plugin in rqt:
+
+```bash
+rqt -s rqt_reconfigure
+```
+
+In the displayed window on the left side choose the node which parameters you
+want to change.
+
+<ImageZoom
+  src="/img/robots/leo/integrations/follow-aruco-marker/aruco-reconfigure.webp"
+  alt="leo rover aruco marker"
+  width="1500"
+  height="1410"
+/>
+
+:::tip
+
+If you don't see your target node, try using the `Refresh` option.
+
+:::
+
+The `aruco_follower` and `aruco_tracker` nodes are the core components that make
+this example work. As their names suggest, one handles marker detection, while
+the other uses that information to command the robot's movement.
+
+#### Tracker
+
+The most important parameter in the `aruco_tracker` node is `marker_size`. If
+your printed marker is not exactly 15 cm, adjust this value accordingly.<br/>
+Other parameters generally don't require any modification, but you're free to
+experiment if you'd like. Keep in mind that once you restart the rover's
+programs, the system will automatically reload the default configuration.
+
+:::warning
+
+Only parameters within the aruco namespace (those named like `aruco.xyz`) are
+safe to modify. Changing anything else may prevent the rover from following the
+marker correctly.<br/> For details on each of those parameters, check the
+[OpenCV ArUco documentation](https://docs.opencv.org/4.6.0/d5/dae/tutorial_aruco_detection.html).
+
+:::
```

Comment:
Why? What about `marker_size`?

Reply:
Isn't this warning suggesting that touching `marker_size` is unsafe as well?

Reply:
Without the text above the note it surely is suggesting that. But as there is info before the warning literally saying to change the parameter value if your marker comply default requirements, I presume it's clear enough.